<?xml version="1.0" ?>
<launch>

  <include file="/home/louis/catkin_ws/src/datmo_master/launch/datmo.launch">
 
  </include>

  <node name="Navigation_node" pkg="navigation" type="Navigation_node" output="screen">
    <param name="cell_radius"     value="3" />
    <param name="goal_tolerance"  value="1.5" />
    <param name="scan_range"      value="20" />
    <!-- Set the dimensions of the cell map here. much to heavt to load everything at once so limit it-->
    <!-- The target waypoint is now at 45 45 so go initialize the map further than that-->
    <param name="x0"              value="-20" />
    <param name="y0"              value="-20" />
    <param name="x1"              value="100" />
    <param name="y1"              value="100" />
  </node>
  
  <node name="simple_dubins_path_nav_node" pkg="coverage_boustrophedon" type="simple_dubins_path_nav_node" output="screen">
    <param name="turning_radius"  value="1.5" />
    <param name="path_resolution" value="0.10" />
  </node>




</launch>
